test/model/adapters/abstract_test.rb

```diff
@@ -78,4 +78,10 @@
       ->{ adapter.transaction({}) }.must_raise NotImplementedError
     end
   end
+
+  describe '#connection_string' do
+    it 'raises error' do
+      ->{ adapter.connection_string({}) }.must_raise Lotus::Model::Adapters::NotSupportedError
```

Comment:
I'll fix it :) I usually do that, but all other lambdas in that file does not have that space there so I thought it was the style of the project. I'll fix the others as well!